sw/airborne/firmwares/rotorcraft/main_chibios.c

```diff
@@ -42,7 +42,7 @@
  * Also include FBW on single MCU
  */
 static void thd_ap(void *arg);
-static THD_WORKING_AREA(wa_thd_ap, 8192);
+THD_WORKING_AREA(wa_thd_ap, 1024);
```

Comment:
again - depending on the autopilot configuration it might not fit in 1k, should be board dependent

Reply:
Here, I don't agree to reduce the stack size. We are running some estimation filters that requires some memory. So I suggest to keep the 8K by default and make it configurable.